### PR TITLE
management plan upload and check

### PR DESCRIPTION
**Pull request by teddmason**

### CODE REVIEW

**Review by reneli5 — APPROVED** [review]

**Review by shaundefra — APPROVED** [review]

File: packages/application-to-register-webapp/src/routes/__tests__/land/upload-management-plan.spec.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        const uploadConfig = Object.assign({}, baseConfig)
+        uploadConfig.filePath = `${mockDataPath}/49MB.pdf`
+        await uploadFile(uploadConfig)
+        setImmediate(() => {
+          done()
+        })
+      })
+    })
+
+    it('should not upload management plan document less than 50 MB', (done) => {
```
[the remark this answers is not in the document]
Comment (teddmason) [reply packages/application-to-register-webapp/src/routes/__tests__/land/upload-management-plan.spec.js:61]: I'll have a look for that when i do another change
